crates/ironrdp-web/src/session.rs

```diff
@@ -861,6 +861,8 @@ fn build_config(
         desktop_scale_factor: 0,
         hardware_id: None,
         license_cache: None,
+        // TODO: implement this
+        vmconnect: None,
```

Comment:
issue: We already support a `pcb` extension. My understanding is that all you need to do is te place a UUID into it. No need to add extra parameters.